types/jvm_types.pp

```diff
@@ -0,0 +1,2 @@
+# @summary Java Virtual Machine (JVM) types
+type Jira::Jvm_types = Enum['openjdk-11', 'oracle-jdk-1.8', 'custom']
```

Comment:
Types also need tests, which seems a bit of pointless effort for three static strings.

Reply:
Reverted it and changed it to simple Enum